radixdlt-core/radixdlt/src/main/java/com/radixdlt/tree/PMT.java

```diff
@@ -101,16 +95,30 @@
   // verify_proof
 
   public PMT(PMTStorage db) {
-    this(db, new SHA256(), new RLPSerializer(), Duration.of(10, ChronoUnit.MINUTES));
+    this(db, (byte[]) null, new SHA256(), new RLPSerializer());
+  }
+
+  public PMT(PMTStorage db, byte[] currentRoot) {
```

Comment:
s/currentRoot/root

Reply:
actuall s/currentRoot/rootHash if we change the constructor to accept a root hash rather than the root node

Reply:
I guess it is fixed with current version